lib/backend/syncersv1/updateprocessors/rules.go

```diff
@@ -116,6 +131,12 @@ func RuleAPIV2ToBackend(ar apiv2.Rule, ns string) model.Rule {
 	}
 }
 
+// parseNamespaceSelector takes a v2 namespace selector and returns the appropriate v1 representation
+// by prefix the keys with the `pcns.` prefix. For example, `k == 'v'` becomes `pcns.k == 'v'`.
+func parseNamespaceSelector(s string) string {
+	return "has(TODO)"
```

Comment:
@fasaxc I'd like to chat with you to figure out the right way to do this. We need to go through the selector and prefix any labels with `pcns.`, then return the modified selector. 

Looks like the lib/selector package and friends should have the right tools to do this.. but I can't quite find the right incantation, so might need some tweaking to make that work.